### ISSUE TITLE
[Exception][IDEA 2023.1.2] java.lang.NoClassDefFoundError: Could not initialize class ai.codium.net.CodiumApiClient

### ISSUE THREAD

**Issue by ostarsier** [Codium-ai/codiumai-jetbrains-release#50]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.lang.NoClassDefFoundError: Could not initialize class ai.codium.net.CodiumApiClient
	at ai.codium.net.CodiumNetworkService.getCodiumApiClient(CodiumNetworkService.kt:147)
	at ai.codium.net.CodiumNetworkService.getMinSupportedVersion(CodiumNetworkService.kt:137)
	at ai.codium.UtilsKt.isUpdateRequired(Utils.kt:108)
	at ai.codium.UtilsKt.isUserLoggedAndUpdateNotRequired(Utils.kt:137)
	at ai.codium.ui.CodiumToolWindowFactory.createToolWindowContent$showDefaultContentByLoggedStatus(CodiumToolWindowFactory.kt:38)
	at ai.codium.ui.CodiumToolWindowFactory.createToolWindowContent(CodiumToolWindowFactory.kt:265)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.createContentIfNeeded(ToolWindowImpl.kt:548)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.scheduleContentInitializationIfNeeded$intellij_platform_ide_impl(ToolWindowImpl.kt:527)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.doShowWindow(ToolWindowManagerImpl.kt:983)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.showToolWindowImpl(ToolWindowManagerImpl.kt:920)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.showToolWindowImpl$default(ToolWindowManagerImpl.kt:906)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.registerToolWindow$intellij_platform_ide_impl(ToolWindowManagerImpl.kt:1094)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.registerToolWindow$intellij_platform_ide_impl$default(ToolWindowManagerImpl.kt:1004)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.initToolWindow$intellij_platform_ide_impl(ToolWindowManagerImpl.kt:532)
	at com.intellij.toolWindow.ToolWindowSetInitializerKt$registerEpListeners$1.extensionAdded(ToolWindowSetInitializer.kt:235)
	at com.intellij.toolWindow.ToolWindowSetInitializerKt$registerEpListeners$1.extensionAdded(ToolWindowSetInitializer.kt:233)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.notifyListeners(ExtensionPointImpl.java:753)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.lambda$registerExtensions$17(ExtensionPointImpl.java:957)
	at com.intellij.ide.plugins.DynamicPlugins.loadPluginWithoutProgress$lambda$29(DynamicPlugins.kt:867)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:958)
	at com.intellij.ide.plugins.DynamicPlugins.loadPluginWithoutProgress(DynamicPlugins.kt:850)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin$lambda$26(DynamicPlugins.kt:822)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.pr

<TRIMMED STACKTRACE>

pplication.impl.ApplicationImpl$3.run(ApplicationImpl.java:456)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:122)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:788)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:731)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:758)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:667)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$7(IdeEventQueue.kt:571)
	at com.intellij.openapi.application.impl.ApplicationImpl.withoutImplicitRead(ApplicationImpl.java:1446)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:571)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:68)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:348)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:348)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:343)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:995)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:995)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$4(IdeEventQueue.kt:343)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:385)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

# Other Information
```
=== About ===
Build version: IntelliJ IDEA 2023.1.2 Build: #IU-231.9011.34 May 16, 2023
Theme: Material Oceanic
JRE: 17.0.6+10-b829.9, JetBrains s.r.o.
JVM: 17.0.6+10-b829.9, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Mac OS X 13.0 (aarch64)

=== Plugins ===
Custom plugins: [Material Theme UI (8.7.1), SonarLint (8.3.0.71062), GitToolBox (500.0.10+213), Atom Material Icons (90.0.0), Codeium: AI Autocomplete for Python, JS, TS, Java, Go, and more (1.2.19), Apifox Helper (1.1.20), CodiumAI - Meaningful tests powered by TestGPT-1 and GPT-3.5&4 (0.5.14), Maven Helper (4.23.222.2964.0)]
Disabled plugins:[]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/TlFFP399) by [Unito](https://www.unito.io)

**Comment by ArtsiomCh** [comment]
Thanks for reporting!
Was it one time issue or you experience it often?
Looks like some occasional internal IDE glitch for me... can't reproduce and figure out why that `NoClassDefFoundError `may happened...

**Comment by ArtsiomCh** [comment]
Thanks again for reporting!
Should be fixed in upcoming 0.5.15
Please feel free to reopen/report if experience it again.
